control/README.md

```diff
@@ -1,6 +1,42 @@
-# Filament Ethereum Outpost
+# Filament Ethereum Control
 
 ## Token grants
 
 The implementation of token grants is a modified version of the
 [starknet token](https://github.com/starknet-io/StarkNet-Token).
+
+## Developement
+
+Install foundry:
+
+```Bash
+curl -L https://foundry.paradigm.xyz | bash
+```
+
```

Comment:
This section should also mention that dependencies need to be installed from within the `control` directory with `forge install`